### PR TITLE
*: stabilize some tests for CI

### LINKED ISSUE #12685: `make dev` test failed in TestInfo
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?
just run `make dev`

2. What did you expect to see?
all tests pass

3. What did you see instead?
domain_test.go:158: start syncer failed

4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?
master

```
[2019-10-14T06:31:40.882Z] [2019/10/14 14:31:34.267 +08:00] [INFO] [domain.go:142] ["full load InfoSchema success"] [usedSchemaVersion=0] [neededSchemaVersion=0] ["start time"=249.993µs]

[2019-10-14T06:31:40.882Z] [2019/10/14 14:31:34.268 +08:00] [WARN] [syncer.go:151] ["[ddl] etcd-cli put kv failed"] [key=/tidb/ddl/all_schema_versions/d088df99-d1bd-4e9f-8121-1556cf14a46c] [value=0] [error="etcdserver: requested lease not found"] [retryCnt=0]

[2019-10-14T06:31:40.882Z] [2019/10/14 14:31:34.298 +08:00] [INFO] [domain.go:111] ["update self version failed"] [usedSchemaVersion=0] [neededSchemaVersion=0] [error="etcdserver: requested lease not found"] [errorVerbose="etcdserver: requested lease not found\ngithub.com/pingcap/errors.AddStack\n\t/go/pkg/mod/github.com/pingcap/errors@v0.11.4/errors.go:174\ngithub.com/pingcap/errors.Trace\n\t/go/pkg/mod/github.com/pingcap/errors@v0.11.4/juju_adaptor.go:15\ngithub.com/pingcap/tidb/ddl/util.PutKVToEtcd\n\t/home/jenkins/workspace/tidb_continue_test/go/src/github.com/pingcap/tidb/ddl/util/syncer.go:154\ngithub.com/pingcap/tidb/ddl/util.(*schemaVersionSyncer).UpdateSelfVersion\n\t/home/jenkins/workspace/tidb_continue_test/go/src/github.com/pingcap/tidb/ddl/util/syncer.go:264\ngithub.com/pingcap/tidb/domain.(*Domain).loadInfoSchema.func1\n\t/home/jenkins/workspace/tidb_continue_test/go/src/github.com/pingcap/tidb/domain/domain.go:109\ngithub.com/pingcap/tidb/domain.(*Domain).loadInfoSchema\n\t/home/jenkins/workspace/tidb_continue_test/go/src/github.com/pingcap/tidb/domain/domain.go:147\ngithub.com/pingcap/tidb/domain.(*Domain).Reload\n\t/home/jenkins/workspace/tidb_continue_test/go/src/github.com/pingcap/tidb/domain/domain.go:342\ngithub.com/pingcap/tidb/domain.(*Domain).loadSchemaInLoop\n\t/home/jenkins/workspace/tidb_continue_test/go/src/github.com/pingcap/tidb/domain/domain.go:454\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1357"]

[2019-10-14T06:31:40.882Z] [2019/10/14 14:31:34.298 +08:00] [INFO] [domain.go:351] ["full load and reset schema validator"]

[2019-10-14T06:31:40.882Z] [2019/10/14 14:31:34.299 +08:00] [INFO] [syncer.go:201] ["close session failed"] [error="etcdserver: requested lease not found"]

[2019-10-14T06:31:40.882Z] [2019/10/14 14:31:34.299 +08:00] [INFO] [domain.go:564] ["domain closed"] ["take time"=137.141137ms]

[2019-10-14T06:31:40.882Z] 2019-10-14 14:31:34.299234 I | integration: terminating 4327105617744525419 (unix://localhost:43271056177445254190)

[2019-10-14T06:31:40.882Z] 2019-10-14 14:31:34.300739 I | integration: terminated 4327105617744525419 (unix://localhost:43271056177445254190)

[2019-10-14T06:31:40.882Z] --- FAIL: TestInfo (2.01s)

[2019-10-14T06:31:40.882Z]     domain_test.go:158: start syncer failed

[2019-10-14T06:31:40.882Z] FAIL
```

### LINKED ISSUE #12942: Unit test case TestSyncerSimple fails randomly
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?

The case `TestSyncerSimple` might not be stable. It sometimes may fail.

```


[2019-10-25T11:48:43.418Z] --- FAIL: TestSyncerSimple (3.02s)

[2019-10-25T11:48:43.418Z]     syncer_test.go:144: update self version failed context deadline exceeded

[2019-10-25T11:48:43.418Z]         github.com/pingcap/errors.AddStack

[2019-10-25T11:48:43.418Z]         	/home/jenkins/agent/workspace/tidb_ghpr_check_2/go/pkg/mod/github.com/pingcap/errors@v0.11.4/errors.go:174

[2019-10-25T11:48:43.418Z]         github.com/pingcap/errors.Trace

[2019-10-25T11:48:43.418Z]         	/home/jenkins/agent/workspace/tidb_ghpr_check_2/go/pkg/mod/github.com/pingcap/errors@v0.11.4/juju_adaptor.go:15

[2019-10-25T11:48:43.418Z]         github.com/pingcap/tidb/ddl/util.PutKVToEtcd

[2019-10-25T11:48:43.418Z]         	/home/jenkins/agent/workspace/tidb_ghpr_check_2/go/src/github.com/pingcap/tidb/ddl/util/syncer.go:154

[2019-10-25T11:48:43.418Z]         github.com/pingcap/tidb/ddl/util.(*schemaVersionSyncer).UpdateSelfVersion

[2019-10-25T11:48:43.419Z]         	/home/jenkins/agent/workspace/tidb_ghpr_check_2/go/src/github.com/pingcap/tidb/ddl/util/syncer.go:264

[2019-10-25T11:48:43.419Z]         github.com/pingcap/tidb/ddl/util_test.TestSyncerSimple

[2019-10-25T11:48:43.419Z]         	/home/jenkins/agent/workspace/tidb_ghpr_check_2/go/src/github.com/pingcap/tidb/ddl/util/syncer_test.go:142

[2019-10-25T11:48:43.419Z]         testing.tRunner

[2019-10-25T11:48:43.419Z]         	/usr/local/go/src/testing/testing.go:909

[2019-10-25T11:48:43.419Z]         runtime.goexit

[2019-10-25T11:48:43.419Z]         	/usr/local/go/src/runtime/asm_amd64.s:1357
```


2. What did you expect to see?



3. What did you see instead?



4. What version of TiDB are you using (`tidb-server -V` or run `select tidb_version();` on TiDB)?

```
mysql> select tidb_version();
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| tidb_version()                                                                                                                                                                                                                                                                                                            |
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Release Version: v4.0.0-alpha-617-g72c5d26
Git Commit Hash: 72c5d26966687c38a157dd25b9386d8811ba47b5
Git Branch: master
UTC Build Time: 2019-10-25 06:55:52
GoVersion: go version go1.13 linux/amd64
Race Enabled: false
TiKV Min Version: v3.0.0-60965b006877ca7234adaced7890d7b029ed1306
Check Table Before Drop: false |
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

### PR DESCRIPTION
**Pull request by wjhuang2016**
### What problem does this PR solve? 

stabilize some tests for CI, Fix #12685, Fix #12942

### What is changed and how it works?

Enlarge timeout

### Check List 

- Tests 

    N/A

### CODE REVIEW

File: ddl/util/syncer_test.go [hunk 1]
```diff
@@ -178,7 +178,7 @@ func TestSyncerSimple(t *testing.T) {
 	if err != nil {
 		t.Fatalf("new session failed %v", err)
 	}
-	childCtx, cancel = context.WithTimeout(ctx, 100*time.Millisecond)
```
Comment (SunRunAway) [inline ddl/util/syncer_test.go]: Do we really need timeout in this test? Could we use `context.Background()`?

Comment (wjhuang2016) [reply ddl/util/syncer_test.go]: I think we can

**Comment by zz-jason** [comment]
@wjhuang2016 should we cherry pick this commit to the release branches?

**Comment by wjhuang2016** [comment]
> @wjhuang2016 should we cherry pick this commit to the release branches?

Yeah

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**